.github/workflows/lint.yml

```diff
@@ -110,21 +112,11 @@ jobs:
           components: rustfmt
       - run: cargo fmt --all --check
 
-  grafana:
```

Comment:
please don't remove this, we use it to ensure formatting

Reply:
I move this into an independent job, launch only if the grafana files are modified